go/json2/marshal_test.go

```diff
@@ -29,11 +32,21 @@ func TestMarshalPB(t *testing.T) {
 		Type: querypb.Type_VARCHAR,
 	}
 	b, err := MarshalPB(col)
-	if err != nil {
-		t.Fatal(err)
-	}
+
+	require.NoError(t, err, "MarshalPB(col) error")
```

Comment:
would probably be more useful to include the actual argument here (i.e. `MarshalPB(%+v)`) and also below

Reply:
Done